usage-advanced.py

```diff
@@ -2,226 +2,215 @@
 
 import dash_player
 import dash
-import dash_html_components as html
-import dash_core_components as dcc
-from dash.dependencies import Input, Output, State
+from dash import dcc, html, Input, Output, State
 
 app = dash.Dash(__name__)
 server = app.server
 
 app.scripts.config.serve_locally = True
```

Comment:
🔪 this is the default now (and can be a `Dash` constructor arg anyway if you want to change it)